modules/osdk-monitoring-custom-metrics.adoc

```diff
@@ -0,0 +1,158 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/osdk-monitoring-prometheus.adoc
+
+[id="osdk-monitoring-custom-metrics_{context}"]
+= Exposing custom metrics
+
+As an Operator author, you can publish custom metrics by using the global Prometheus registry from the `controller-runtime/pkg/metrics` library.
+
+.Prerequisites
+
+* Go-based Operator generated using the Operator SDK
+* {product-title} cluster, which deploys the Prometheus Operator by default
```

Comment:
`, which deploys the Prometheus Operator by default`

I'm not sure whether this phrase describes OCP or it's telling me I need to set this as a default.

Reply:
The former, I'll re-write.